man/fi_domain.3.md

```diff
@@ -666,6 +667,78 @@ Applications can set the mr_cnt on input to fi_getinfo, in order to
 provider to optimize any memory registration cache or lookup tables.
 
+## Traffic Class (tclass)
+
+The default traffic class associated with the domain and any endpoints
+created within the context of the domain.
+
+Applications can set the tclass in the domain, in order to indicate
+a request for a specific default traffic class. Otherwise, if the application sets
+the tclass field to FI_TC_TYPE_UNSPEC, the provider will set the
+default traffic class for the domain to be FI_TC_BEST_EFFORT.
+
+Traffic classes can be set according to two different types of classifications
+available to libfabric: FI_TC_TYPE_LABEL or FI_TC_TYPE_DSCP. If the traffic class
+*type* field is set to FI_TC_TYPE_DSCP, the value field of the traffic class must be set
+to a DSCP value. If the traffic class *type* field is set to FI_TC_TYPE_LABEL, the
+value field of the traffic class must be set to a libfabric or provider-specific
+defined label of a traffic class.
+
+Traffic classes can be any of the following defined classes of traffic,
+or any provider-specific definitions.
+
+** FI_TC_BEST_EFFORT **
+** FI_TC_LOW_LATENCY **
+** FI_TC_DEDICATED_ACCESS **
+** FI_TC_BULK_IO_DATA **
+** FI_TC_SCAVENGER **
+** FI_TC_MANAGEMENT **
+** FI_TC_BULK_IPC_DATA **
+
+The general behavior provided for each traffic class is provided below.
+
+*FI_TC_BEST_EFFORT*
+: This is the default, shared class, carrying the traffic for a number of
+applications executing concurrently over the same network infrastructure.
+Even though it is shared, network capacity and resource allocation are
+distributed fairly across the applications.
+
+*FI_TC_LOW_LATENCY*
+: This class supports low latency, low jitter data patterns typically caused by
+transactional data exchanges, barrier synchronizations, and collective operations
+that are typical of HPC applications. Specified maximum latencies are requested
+by this class.  Fulfillment of such requests in this class will typically require
+accompanying bandwidth caps and packet size limitations so as not to consume
+excessive bandwidth at high priority.
+
+*FI_TC_DEDICATED_ACCESS*
+: This class operates at the highest priority, except the management class.
+It carries a high bandwidth allocation, maximum latency targets, and the
+highest scheduling and arbitration priority.
+
+*FI_TC_BULK_IO_DATA*
+: This class is intended for large data transfers associated with I/O and
+is present to separate sustained I/O transfers from other application
+inter-process communications.
+
+*FI_TC_SCAVENGER*
+: This class is used for data that is desired but does not have strict delivery
+requirements such as monitoring data especially for application-specific
+monitoring such as with performance tools.  Global monitoring could also be done
+this way for data that is too voluminous for out-of-band transport.  Using this
+class makes sure that such communication does not interfere with communication doing “real” work.
```

Comment:
Please watch line lengths and try to limit to 80 characters or less.  For the format, indent all text, for example:
```
: This is the start of some description.  This is the
  continuation of that description.  The entire paragraph
  should be indented similarly.